readthedocs/templates/projects/domain_list.html

```diff
@@ -23,7 +23,7 @@
     {% for domain in object_list %}
         <li {% if domain.machine %}class="help_text"{% endif %}>
         {% if domain.canonical %}<strong>{% endif %}
-        {{ domain.url }}
+        {{ domain.domain }}
```

Comment:
Might as well define this as a more semantic css selector, ie - `domain-selected`. Specific formatting should be kept out of the html and push to css.